clients/go/apis/v1alpha1/config.go

```diff
@@ -358,3 +349,24 @@ func (r *AuditRule) SetDefault() {
 		r.LogType = AuditLogRequest_DATA_ACCESS.String()
 	}
 }
+
+// Justification specifies the config used to integrate with JVS.
+type Justification struct {
+	// JVSPublicKeysEndpoint is the endpoint where public keys may be retrieved from the JVS.
+	// These will be used to validate JWT tokens that are passed in through the
+	// "justification-token" header.
+	JVSPublicKeysEndpoint string `yaml:"jvs_public_keys_endpoint,omitempty"`
```

Comment:
I think JVS here is redundant info. Should we remove it?

Reply:
done